tools/web_features/manifest.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import os
+
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, List, Optional, Set
+
+from ..manifest.item import SupportFile
+from ..manifest.sourcefile import SourceFile
+from ..metadata.yaml.load import load_data_to_dict
+from ..web_features.web_feature_map import WebFeatureToTestsDirMapper, WebFeaturesMap
+from .. import localpaths
+from ..metadata.webfeatures.schema import WEB_FEATURES_YML_FILENAME, WebFeaturesFile
+
+"""
+This command generates a manifest file. This file contains a mapping of web
+features to manifest items. The data comes from the WEB_FEATURES.yml that are
+throughout the wpt source code. For more details about the file, check out RFC 163
+https://github.com/web-platform-tests/rfcs/pull/163.
+
+The file written is a JSON file. The format is
+{
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "$ref":"#/definitions/File",
+   "definitions":{
+      "File":{
+         "type":"object",
+         "additionalProperties":false,
+         "properties":{
+            "version":{
+               "type":"integer",
+               "description":"Schema version of the file.",
+               "enum":[
+                  1
+               ]
+            },
+            "data":{
+               "type":"object",
+               "description":"High level container for the data. Object key is the web feature.",
+               "additionalProperties":{
+                  "type":"array",
+                  "items":{
+                     "$ref":"#/definitions/ManifestItem"
+                  }
+               }
+            },
+            "config":{
+               "$ref":"#/definitions/Config"
+            }
+         },
+         "required":[
+            "config",
+            "data",
+            "version"
+         ],
+         "title":"File"
+      },
+      "Config":{
+         "type":"object",
+         "description":"Information about config to generate the manifest file",
+         "additionalProperties":false,
+         "properties":{
+            "url_base":{
+               "type":"string"
+            }
+         },
+         "required":[
+            "url_base"
+         ],
+         "title":"Config"
+      },
+      "ManifestItem":{
+         "type":"object",
+         "additionalProperties":false,
+         "properties":{
+            "url":{
```

Comment:
This is a (relative) URL, but I wonder if "path" would work here? It seems like the more common terminology I've heard people use when referring to these strings, and matches wpt.fyi's `path:/.../` query syntax, for example.

Reply:
I agree that `path` would be better. However, the variable name comes from the existing `url` field in URLManifestItem [1] [2]. As a result, I thought it would be better to keep it consistent. Let me know your thoughts.

[1] The original value:
- https://github.com/web-platform-tests/wpt/blob/d4d9e95b565eadcd672aeab45b70dbd924020f6c/tools/manifest/item.py#L112-L117

[2] Where we copy the value
- https://github.com/web-platform-tests/wpt/pull/43828/files#diff-3a95149a0e08f798d0a1b597ed2d0b9e4477f82192fb30addacdc46b958daf52R16-R18

Reply:
Both would be OK, but I still think we should go with `path`, in particular because of the wpt.fyi search syntax.